### PR TITLE
Update grant-permissions script

### PR DESCRIPTION
**Pull request by mrmemes-eth**
Takes the private key from forge script instead of explicitly passing it.

### CODE REVIEW

File: bin/jib [hunk 1]
```diff
@@ -228,12 +228,11 @@ checkStorage() {
 # @option --gas-price Override the gas price to use for the transaction.
 grantPermissions() {
   ADDRESS_ARRAY=$(awk 'BEGIN { printf "[" } { if (NR == 1) printf "%s", $0; else printf ",%s", $0; } END { printf "]" }' $argc_file)
-  PRIVATE_KEY=$argc_private_key forge script \
-    script/Util.s.sol:GrantPermissions \
+  forge script script/Util.s.sol:GrantPermissions \
     $argc_contract $ADDRESS_ARRAY \
     --sig "run(address,address[])" \
     --rpc-url $argc_rpc_url --broadcast -vvvv \
-    --gas-price "${argc_gas_price:-0}"
```
Comment (desi) [inline bin/jib:236]: Is this removal meant for this PR?

Comment (mrmemes-eth) [reply bin/jib:236]: I was doing some drive-by cleanup... that was a shim to compensate for gas prices on Polygon being really high and us having transactions fail repeatedly. I would be fine with reverting that here if you think it's inappropriate.

**Review by desi — APPROVED** [review]
LGTM

File: bin/jib (hunk 1 above)
Comment (mrmemes-eth) [reply bin/jib:236]: No, hopefully it's not necessary in general. If we did need to add it back though, I would rather make it a global option for all scripts than just on this `grant-permissions` script.

**Pull request merged**